File: Data Structures & Algorithms/max-consecutive-ones/submission-2.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int maxCount = 0;
        int count = 0;
        for (int i = 0; i < nums.length; i++) {
            count = nums[i] == 1 ? count + 1 : 0;
            maxCount = Math.max(maxCount, count);
        }
        return maxCount;
    }
}

File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-2.py
class Solution:
    def replaceElements(self, arr: List[int]) -> List[int]:
        m = -1
        for i in range(len(arr) - 1, -1, -1):
            val = arr[i]
            arr[i] = m
            m = max(m, val)
        return arr
